x-pack/plugins/cross_cluster_replication/public/app/components/follower_index_form/follower_index_form.js

```diff
@@ -489,8 +533,7 @@ export const FollowerIndexForm = injectI18n(
                   <p>
                     <FormattedMessage
                       id="xpack.crossClusterReplication.followerIndexForm.advancedSettingsDescription"
-                      defaultMessage="Customize advanced settings to control the rate at which data is replicated.
-                        If you don't customize them, default advanced settings will be applied."
+                      defaultMessage="Advanced settings control the rate of replication."
```

Comment:
are we ok with dropping the copy about default settings?

Reply:
How about:

Advanced settings control the rate at which data is replicated.  You can customize these settings or use the default values.

Reply:
👍 I left out the mention of "data" per @jen-huang's previous comments:

> Advanced settings control the rate of replication. You can customize these settings or use the default values.